posts/inside-rust/2022-06-22-survey-2021-report.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: 2021 Annual Survey Report
+author: Nick Cameron
+description: "Download a data report on the 2021 annual community survey."
+team: The Survey Working Group <https://www.rust-lang.org/governance/teams/community#Survey%20team>
+---
+As usual, we conducted an annual community survey in 2021. We previously shared some some highlights and charts
+in a [blog post](https://blog.rust-lang.org/2022/02/15/Rust-Survey-2021.html). This year we would also like to
+make the complete (-ish) dataset available. We have compiled a report which contains data and charts for nearly
+all questions with minimal analysis. We have elided a few sensitive questions and have combined some answers
+or elided some answers where there is any chance of respondents being identified or of sensitive data being released.
+
+We intend to produce further small reports with more analysis targetted at specific teams or groups within the
+project. If there is any analysis or processed data you'd like to see, please get in touch with the survey working group via Zulip.
```

Comment:
Let's link to the [Zulip stream for the survey working group](https://rust-lang.zulipchat.com/#narrow/stream/294169-t-community.2Frust-survey-2021).